app/scripts/vueCore.js

```diff
@@ -131,6 +133,114 @@
              };
              this.charts = {};
              },*/
+            openCases:function(){
+              iViz.openCases();
+              
+             /* var studyId = '';
```

Comment:
Can we remove these commented out code? I think they have been isolated into an individual function?